src/tmpfiles/tmpfiles.c

```diff
@@ -1634,6 +1609,29 @@ static int create_subvolume(Item *i, const char *path) {
         return r;
 }
 
+static int empty_directory(Item *i, const char *path) {
+        int r;
+
+        assert(i);
+        assert(i->type == EMPTY_DIRECTORY);
+
+        r = is_dir(path, false);
+        if (r == -ENOENT) {
+                /* Option "e" operates only on existing objects. Do not
+                 * print errors about non-existent files or directories */
+                log_debug("Skipping missing file: %s", path);
+                return 0;
+        }
+        if (r < 0)
+                return log_error_errno(r, "is_dir() failed on file %s: %m", path);
```

Comment:
`failed on path "\%s\"` or `failed on directory "\%s\"`

Reply:
"failed on path" seems more appropriate since is_dir() could have failed on a file, directory or whatever at this point.